uplinks/freifunk-berlin-uplink-tunnelberlin-tunneldigger-files/uci-defaults/freifunk-berlin-z95_tunnelberlin-tunneldigger

```diff
@@ -27,17 +24,9 @@ uci set network.ffuplink_dev.name=ffuplink
 # Create a static random macaddr for ffuplink device
 macaddr=$(macaddr_canonicalize `dd 2>/dev/null bs=1 count=6 </dev/urandom | hexdump -e '1/1 "%02x"'`)
```

Comment:
This macaddr creation line is problematic.  I have been trying and trying to change this back to a for-loop everywhere, but this code keeps showing it's face.  See 80cb77d5d400f4c737f3ee4260bee800cd6057d1

Reply:
that's the code we currently have on master for this. This PR only deals with the points it's meat to deal with as per PR-message, so it will not change anything unrelated.